src/CoreBundle/Bridge/Chameleon/Module/Sidebar/TableMenuItemProvider.php

```diff
@@ -33,7 +33,8 @@ public function createMenuItem(\TdbCmsMenuItem $menuItem): ?MenuItem
             $menuItem->id,
             $menuItem->fieldName,
             $menuItem->fieldIconFontCssClass,
-            PATH_CMS_CONTROLLER."?pagedef=tablemanager&id={$tableConf->id}&_histid=0&_rmhist=true"
+            PATH_CMS_CONTROLLER."?pagedef=tablemanager&id={$tableConf->id}",
```

Comment:
Did you remove "&_histid=0&_rmhist=true"  on purpose?

Reply:
I wasn't actually the author, but it looks as the MTTableManager class does not process these dropped parameters at all.